internal/benchrunner/reporters/formats/json.go

```diff
@@ -5,54 +5,53 @@
 package formats
 
 import (
-	"encoding/xml"
-
-	"github.com/pkg/errors"
+	"encoding/json"
+	"fmt"
 
 	"github.com/elastic/elastic-package/internal/benchrunner"
 )
 
 func init() {
-	benchrunner.RegisterReporterFormat(ReportFormatXUnit, reportXUnitFormat)
+	benchrunner.RegisterReporterFormat(ReportFormatJSON, jsonFormat)
 }
 
 const (
-	// ReportFormatXUnit reports benchmark results in the xUnit format
-	ReportFormatXUnit benchrunner.BenchReportFormat = "xUnit"
```

Comment:
Would it make sense to keep support for both formats? xunit and json? In case we use some tool or plugin in the future that can consume the xunit format. It looks like same structs could be reused, with the annotations for both formats.